server/controllers/api/claim/data/index.js

```diff
@@ -11,7 +13,14 @@ const claimData = ({ ip, originalUrl, body, params }, res) => {
   const claimName = params.claimName;
   let claimId = params.claimId;
   if (claimId === 'none') claimId = null;
-  db.Claim.resolveClaim(claimName, claimId)
+  chainquery.claim.queries.resolveClaim(claimName, claimId)
+    .then(claimInfo => {
+      if (!claimInfo) {
+        // Not found remote, try local
+        return db.Claim.resolveClaim(claimName, claimId)
```

Comment:
I like the backup for now. Good for resilience for intial cutover.

Reply:
It's also, unfortunately, required since we don't wait for `chainquery` to have the claim before rendering the content page.  We have a number of nuances that require the fallback and prevent us from fully dropping tables.